cni-plugin/pkg/install/install.go

```diff
@@ -196,14 +197,20 @@ func Install() error {
 			continue
 		}
 
+		containerBinDir := "/opt/cni/bin/"
+		// The binaries dir in the container needs to be prepended by the CONTAINER_SANDBOX_MOUNT_POINT env var on Windows Host Process Containers
+		// see https://kubernetes.io/docs/tasks/configure-pod-container/create-hostprocess-pod/#containerd-v1-7-and-greater
+		if runtime.GOOS == "windows" {
+			containerBinDir = os.Getenv("CONTAINER_SANDBOX_MOUNT_POINT") + "/" + containerBinDir
```

Comment:
@jsturtevant I still needed to use the `$CONTAINER_SANDBOX_MOUNT_POINT` env var when referring to the binaries inside the container here, which from what I read on https://kubernetes.io/docs/tasks/configure-pod-container/create-hostprocess-pod/#containerd-v1-7-and-greater shouldn't be needed, right? Could you please take a look an let me know if I missed something?

Reply:
what does the yaml look like for your deployment? I don't see it in the PR here.  Could it be that the Working directory is set to something?

Reply:
I've mostly been working with the tigera operator with the changes from this PR: https://github.com/tigera/operator/pull/2732, but the deployment also works with this manifest: https://gist.github.com/coutinhop/7528503be5d0aeb22001d69d82fa61ed

No working dir on the install-cni initContainer, but I did include `$env:CONTAINER_SANDBOX_MOUNT_POINT` in the path for `command` in it, could that cause issues?

Reply:
> No working dir on the install-cni initContainer, but I did include `$env:CONTAINER_SANDBOX_MOUNT_POINT` in the path for `command` in it, could that cause issues?

yes I think that would do it.  could you try with something like `command: ["opt/cni/bin/install.exe"]` or `/opt/cni/bin/install.exe` depending on where those files live?

Reply:
Thanks! I've tried it (removing mentions to `$env:CONTAINER_SANDBOX_MOUNT_POINT`), but I got
```
'/opt/cni/bin/install.exe' is not recognized as an internal or external command
```

I see that the yaml for kube-proxy also still uses `$env:CONTAINER_SANDBOX_MOUNT_POINT`
https://github.com/kubernetes-sigs/sig-windows-tools/blob/3ed28295f8694dffd03e144af4423137b2b8c30b/hostprocess/calico/kube-proxy/kube-proxy.yml#L25

Apparently the binaries (and scripts, I couldn't manage to make the uninstall-calico initContainer work either) are not in these absolute paths, I'm guessing some path in `c:/hpc/...` would work, but then why not just use `$env:CONTAINER_SANDBOX_MOUNT_POINT`, right?

This is by far not a deal breaker, stuff was working so I wouldn't block anything by it (unless you see `$env:CONTAINER_SANDBOX_MOUNT_POINT` being dropped at some point and these containers as they are becoming unsupported).

One other quick question: is the presence of `$env:CONTAINER_SANDBOX_MOUNT_POINT` a reliable indicator that we're inside a host process container? We've used it in a couple of places, I wonder if we should look into changing that...

Once again, thanks so much!